tests/phpunit/Unit/Graph/GraphFormatterTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace SRF\Tests\Unit\Formats;
+
+use SRF\Graph\GraphFormatter;
+use SRF\Graph\GraphNode;
+
+
+/**
+ * @covers \SRF\Graph\GraphFormatter
+ * @group semantic-result-formats
+ *
+ * @license GNU GPL v2+
+ *
+ * @author Sebastian Schmid (gesinn.it)
+ */
+class GraphFormatterTest extends \PHPUnit_Framework_TestCase {
+
+	/*
+	* @see https://www.semantic-mediawiki.org/wiki/Help:Graph_format
+	*/
+	private $options = [
+		'graphName' => "Unit Test",
+		'graphSize' => "100",
+		'nodeShape' => "rect",
+		'nodeLabel' => "displaytitle",
+		'rankDir' => "TB",
+		'wordWrapLimit' => "20",
+		'parentRelation' => "parent",
+		'enableGraphLink' => "yes",
+		'showGraphLabel' => "yes",
+		'showGraphColor' => "yes",
+		'showGraphLegend' => "yes",
+	];
+
+	private $graphFormatter;
+
+	private $nodes = [];
+
+	protected function setUp() {
+		parent::setUp();
+
+		$this->graphFormatter = new GraphFormatter( $this->options );
+
+		$this->nodes = [];
+
+		$node1 = new GraphNode( 'Team:Alpha' );
+		$node1->setLabel( "Alpha" );
+		$node1->addParentNode( "Casted", "Person:Alexander Gesinn" );
+		$this->nodes[] = $node1;
+
+		$node2 = new GraphNode( 'Team:Beta' );
+		$node2->setLabel( "Beta" );
+		$node2->addParentNode( "Casted", "Person:Sebastian Schmid" );
+		$node2->addParentNode( "Casted", "Person:Alexander Gesinn" );
+		$node2->addParentNode( "Part of Team ", "Team:Alpha" );
+		$this->nodes[] = $node2;
+
+		$this->graphFormatter->buildGraph( $this->nodes );
```

Comment:
Looks like there is no need to have class level state here. You could just have a newGarphFormatter method that you call when you need it.